supplementary/alica_templates/Expr/src/ActionServerExample2379894799421542548.cpp

```diff
@@ -0,0 +1,137 @@
+#include "ActionServerExample2379894799421542548.h"
+/*PROTECTED REGION ID(eph2379894799421542548) ENABLED START*/
+// Add additional options here
+#include <optional>
+#include <string>
+/*PROTECTED REGION END*/
+
+namespace alica
+{
+// Plan:  ActionServerExample (2379894799421542548)
+//
+// Tasks:
+//   - DefaultTask (1225112227903) (Entrypoint: 1647616282106629095)
+//
+// States:
+//   - WaitForGoal (4209576477302433246)
+//   - ExecuteGoal (2119574391126023630)
+ActionServerExample2379894799421542548::ActionServerExample2379894799421542548(IAlicaWorldModel* wm)
+        : DomainPlan(wm)
+{
+    /*PROTECTED REGION ID(con2379894799421542548) ENABLED START*/
+    // Add additional options here
+    _actionServer = std::make_unique<actionlib::SimpleActionServer<alica_templates::DummyAction>>(_nh, std::string("DummyActionServer"), false);
+    _actionServer->start();
```

Comment:
Ideally we should create and destroy this in plan init and terminate. And if you set last argument (auto_start) as true, you don't need to explicitly start the server.

Reply:
won't this cause more overhead in establishing the connection each time the action server is destroyed & then recreated?